City-of-Helsinki/drupal-helfi
# UHF-1859 Banner to basic page, unit and service

How to test with local setup:
- Switch to corresponding branches: 
  - `git fetch`
  - `git checkout UHF-1859-banner-to-basic-page-unit-service`
- `composer require drupal/hdbt:dev-UHF-1859-banner-to-basic-page-unit-service && composer require drupal/helfi_platform_config:dev-UHF-1859-banner-to-basic-page-unit-service`
- Run `make drush-cim`
- Run `make drush-cr`
- Log in and go to edit one basic page, one service and one unit.
- Make sure you can add banners for both upper and lower content on all of the content types mentioned above.
- Save the pages and make sure that banners look good and scale in a correct way. They should work the same way as they do in the landing page.

Also check these:
https://github.com/City-of-Helsinki/drupal-helfi-sote/pull/50
https://github.com/City-of-Helsinki/drupal-helfi-kymp/pull/41
https://github.com/City-of-Helsinki/drupal-hdbt/pull/129
https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/133
